onnx/test/cpp/inliner_test.cc

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) ONNX Project Contributors
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <iostream>
+#include "gtest/gtest.h"
+#include "onnx/checker.h"
+#include "onnx/common/constants.h"
+#include "onnx/defs/parser.h"
+#include "onnx/defs/printer.h"
+#include "onnx/defs/schema.h"
+#include "onnx/inliner/inliner.h"
+#include "onnx/shape_inference/implementation.h"
+
+namespace ONNX_NAMESPACE {
+namespace Test {
+
+static void InlineFunctions(ModelProto& model, const char* input) {
+  OnnxParser parser(input);
+  auto status = parser.Parse(model);
+  EXPECT_TRUE(status.IsOK()) << status.ErrorMessage();
+  EXPECT_TRUE(parser.EndOfInput()) << "Extra unparsed input unexpected.";
+
+  checker::check_model(model);
+  shape_inference::InferShapes(model);
+
+  std::cout << ProtoToString(model) << "\n";
```

Comment:
Is it expected in a test to print something?

Reply:
No, that was just for my own local validation. Will remove or comment it out, thanks for catching it.